README.md

```diff
@@ -59,6 +41,10 @@ The project provides several API endpoints that allow you to interact with the f
 - It may not get the intention of your instructions right away. It's meant to be a conversational tool.
 - If the assistant starts hallucinating, it may be helpful to start a new conversation or limit the length of each file being loaded.
 
+## API Endpoints
+
+View API endpoints that allow you to interact with the file system within the Kaguya directory [here](pages/apiEndpoints.md).  
```

Comment:
We should specify that these interact with the FILES directory by default

Reply:
OK will add.